src/frontend/js/widgets/Dashboard/components/DashboardItem/Enrollment/DashboardItemEnrollment.spec.tsx

```diff
@@ -20,11 +44,7 @@ describe('<DashboardItemEnrollment/>', () => {
     }).one();
     enrollment.course_run.state.priority = Priority.ONGOING_OPEN;
 
-    render(
-      <IntlProvider locale="en">
-        <DashboardItemEnrollment enrollment={enrollment} />
-      </IntlProvider>,
-    );
+    render(Wrapper({ enrollment }));
```

Comment:
Not really "react" style: 

```suggestion
render(<Wrapper enrollment={enrollment} />)
```